compiler/lsp-tests/src/Main.hs

```diff
@@ -646,3 +649,76 @@ regressionTests run _runScenarios = testGroup "regression"
           , _command = Nothing
           , _xdata = Nothing
           }
+
+multiPackageTests :: FilePath -> TestTree
+multiPackageTests damlc = testGroup "multi-package"
+    [ testCaseSteps "example 1" $ \step -> withTempDir $ \dir -> do
+          step "build a"
+          createDirectoryIfMissing True (dir </> "a")
+          writeFileUTF8 (dir </> "a" </> "daml.yaml") $ unlines
+              [ "sdk-version: ignored"
+              , "name: a"
+              , "version: 0.0.1"
+              , "source: ."
+              , "dependencies: [daml-prim, daml-stdlib]"
+              ]
+          writeFileUTF8 (dir </> "a" </> "A.daml") $ unlines
+              [ "daml 1.2 module A where"
+              , "data A = A"
+              , "a = A"
+              ]
+          withCurrentDirectory (dir </> "a") $ callProcess damlc ["build", "-o", dir </> "a" </> "a.dar"]
+          step "build b"
+          createDirectoryIfMissing True (dir </> "b")
+          writeFileUTF8 (dir </> "b" </> "daml.yaml") $ unlines
+              [ "sdk-version: ignored"
+              , "name: b"
+              , "version: 0.0.1"
+              , "source: ."
+              , "dependencies: [daml-prim, daml-stdlib, " <> show (".." </> "a" </> "a.dar") <> "]"
+              ]
+          writeFileUTF8 (dir </> "b" </> "B.daml") $ unlines
+              [ "daml 1.2 module B where"
+              , "import A"
+              , "f : Scenario A"
+              , "f = pure a"
+              ]
+          withCurrentDirectory (dir </> "b") $ callProcess damlc ["build", "-o", dir </> "b" </> "b.dar"]
+          step "run language server"
+          writeFileUTF8 (dir </> "daml.yaml") $ unlines
+              [ "sdk-version: ignored"
```

Comment:
🤔 

Why do we need a daml.yaml if it does not have a valid sdk-version field?

Reply:
Because the config file is read by `damlc` not the assistant. More generally, this is what we’re telling users to use so it’s helpful to test. E.g., without the change to `initPackageDb` which only considers a project a project if it has also `name`, …  fields `daml init` would fail if you have a `daml.yaml` with only an `sdk-version`.

Reply:
I’ve changed it to use the current SDK version to make it a bit less confusing.

Reply:
I see, thanks for the explanation. I think using the current SDK version is good. :)